src/Entity/CitizenProject.php

```diff
@@ -153,15 +153,10 @@ class CitizenProject extends BaseGroup implements CoordinatorAreaInterface
     /**
      * @var string|null
      *
-     * @ORM\Column(type="text", nullable=true)
+     * @ORM\Column(length=255, nullable=true)
```

Comment:
To avoid a 500 you should trim the name when needed.

Reply:
The `imageName` is hashed in the setter. Do you think is really needed to trim it ? The name will always be the same size (with the extension .jpg or .png).